NetKet/Machine/machine.hpp

```diff
@@ -41,21 +42,18 @@ class Machine : public AbstractMachine<T> {
 
   explicit Machine(const Hilbert &hilbert, const json &pars)
       : hilbert_(hilbert) {
-    CheckInput(pars);
     Init(hilbert_, pars);
```

Comment:
There should be a CheckInput(pars); for all the constructors

Reply:
I remembered we moved the CheckInput into Init.